app/views/deals/new.html.erb

```diff
@@ -0,0 +1,16 @@
+<%= form_with model: @deal, url: user_deals_path, method: :post do |form| %>
+
+    <p><%= link_to "Back to Deals page", user_deals_path, class:"btn btn-primary" %></p>
+
```

Comment:
- [ ] "Add a new transaction" page should have a button that takes to the transactions page. The button that you added takes the user to the `all_deals` page, not the deal page.